content/en/synthetics/private_locations/_index.md

```diff
@@ -707,6 +707,8 @@ To upgrade an existing private location, click the **Gear** icon on the private
 Then, run the [configuration command based on your environment](#install-your-private-location
 ) to get the latest version of the private location image. 
 
+***Note***: If you're using `docker run` to launch your Private Location image and you've previously installed the Private Location image using the `latest` tag, make sure to add `--pull=always` to the `docker run` command to make sure the newest version is pulled rather than relying on the cached version of the image that may exist locally with the same `latest` tag.
```

Comment:
```suggestion
**Note**: If you're using `docker run` to launch your Private Location image and you've previously installed the Private Location image using the `latest` tag, make sure to add `--pull=always` to the `docker run` command to make sure the newest version is pulled rather than relying on the cached version of the image that may exist locally with the same `latest` tag.
```